src/pages/billing/List.js

```diff
@@ -401,15 +399,19 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
         loadAccounts,
         onDismiss: () => setCreatingBillingProject(null),
         onSuccess: billingProjectName => {
-          Ajax().Metrics.captureEvent(Events.billingCreationGCPBillingProjectCreated, { billingProject: billingProjectName })
+          Ajax().Metrics.captureEvent(Events.billingCreationGCPBillingProjectCreated, {
+            billingProjectName, billingProjectCloudPlatform: cloudProviders.gcp.label
+          })
           setCreatingBillingProject(null)
           loadProjects()
         }
       }),
       creatingBillingProject === cloudProviders.azure && isAlphaAzureUser && h(CreateAzureBillingProjectModal, {
         onDismiss: () => setCreatingBillingProject(null),
         onSuccess: billingProjectName => {
-          Ajax().Metrics.captureEvent(Events.billingCreationAzureBillingProjectCreated, { billingProject: billingProjectName })
+          Ajax().Metrics.captureEvent(Events.billingCreationAzureBillingProjectCreated, {
+            billingProjectName, billingProjectCloudPlatform: cloudProviders.azure.label
```

Comment:
Changed to billingProjectName for consistency. Note that the event name already says Azure… perhaps we should consolidate into a single create billing project name?

![image](https://user-images.githubusercontent.com/484484/209217242-6dd51b3d-bc80-4f97-b5af-45fb0e6ede57.png)